src/Options.php

```diff
@@ -170,29 +170,28 @@ public function get_all() {
 	/**
 	 * Get all the options for a group.
 	 *
-	 * Options::init()->get_group('smtp') - will return only array of options (or empty array if a key doesn't exist).
+	 * Options::init()->get_group('smtp') - will return the array of options, including defaults and constants.
 	 *
 	 * @since 1.0.0
 	 *
 	 * @param string $group
 	 *
-	 * @return mixed
+	 * @return array
 	 */
 	public function get_group( $group ) {
 
 		// Just to feel safe.
 		$group = sanitize_key( $group );
 
-		if ( isset( $this->_options[ $group ] ) ) {
-
-			foreach ( $this->_options[ $group ] as $g_key => $g_value ) {
-				$options[ $group ][ $g_key ] = $this->get( $group, $g_key );
-			}
+		if ( ! isset( $this->_options[ $group ] ) ) {
+			$this->_options[ $group ] = array();
```

Comment:
Do not directly modify default class property `_options`, it's for retrieval only. Modifying will lead to errors. Use `$options` here from above.